MODULE.bazel

```diff
@@ -373,6 +397,44 @@ dev_maven.install(
     lock_file = "//tests/custom_maven_install:multiple_lock_files_install.json",
 )
 
+dev_maven.install(
```

Comment:
I saw this when testing on the Bazel project:

```
    https://repo1.maven.org/maven2/org/checkerframework/checker-qual/3.19.0/checker-qual-3.19.0.pomcom.github.bazelbuild.rules_jvm_external.resolver.remote.UriNotFoundException: Unable to download com.google.code.findbugs:jsr305:3.0.2 from any of [https://repo1.maven.org/maven2]. Required because: com.google.guava:guava-testlib:33.0.0-jre -> com.google.code.findbugs:jsr305:3.0.2
	at com.github.bazelbuild.rules_jvm_external.resolver.cmd.Main.lambda$getInfos$1(Main.java:177)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

Reply:
...and another (on a 96 core machine):

```
java.lang.RuntimeException: org.eclipse.aether.collection.DependencyCollectionException: Failed to collect dependencies at com.google.errorprone:error_prone_check_api:jar:2.23.0 -> com.google.inject:guice:jar:5.1.0
	at com.github.bazelbuild.rules_jvm_external.resolver.maven.MavenResolver.resolveBaseDependencies(MavenResolver.java:382)
	at com.github.bazelbuild.rules_jvm_external.resolver.maven.MavenResolver.resolve(MavenResolver.java:168)
	at com.github.bazelbuild.rules_jvm_external.resolver.cmd.Main.main(Main.java:61)
Caused by: org.eclipse.aether.collection.DependencyCollectionException: Failed to collect dependencies at com.google.errorprone:error_prone_check_api:jar:2.23.0 -> com.google.inject:guice:jar:5.1.0
	at org.eclipse.aether.internal.impl.collect.DependencyCollectorDelegate.collectDependencies(DependencyCollectorDelegate.java:260)
	at org.eclipse.aether.internal.impl.collect.DefaultDependencyCollector.collectDependencies(DefaultDependencyCollector.java:87)
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.collectDependencies(DefaultRepositorySystem.java:306)
	at com.github.bazelbuild.rules_jvm_external.resolver.maven.MavenResolver.resolveBaseDependencies(MavenResolver.java:380)
	... 2 more
Caused by: org.eclipse.aether.resolution.ArtifactDescriptorException: Failed to read artifact descriptor for com.google.inject:guice:jar:5.1.0
	at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.loadPom(DefaultArtifactDescriptorReader.java:289)
	at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.readArtifactDescriptor(DefaultArtifactDescriptorReader.java:175)
	at org.eclipse.aether.internal.impl.collect.bf.BfDependencyCollector.resolveCachedArtifactDescriptor(BfDependencyCollector.java:464)
	at org.eclipse.aether.internal.impl.collect.bf.BfDependencyCollector.resolveDescriptorForVersion(BfDependencyCollector.java:450)
	at org.eclipse.aether.internal.impl.collect.bf.BfDependencyCollector.lambda$resolveArtifactDescriptorAsync$1(BfDependencyCollector.java:417)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
	at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658)
	at org.eclipse.aether.internal.impl.collect.bf.BfDependencyCollector.lambda$resolveArtifactDescriptorAsync$4(BfDependencyCollector.java:416)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.apache.maven.model.resolution.UnresolvableModelException: The following artifacts could not be resolved: com.google.inject:guice-parent:pom:5.1.0 (absent): Could not transfer artifact com.google.inject:guice-parent:pom:5.1.0 from/to https://repo1.maven.org/maven2 (https://repo1.maven.org/maven2): Connect to repo1.maven.org:443 [repo1.maven.org/199.232.192.209, repo1.maven.org/199.232.196.209, repo1.maven.org/2a04:4e42:4c:0:0:0:0:209, repo1.maven.org/2a04:4e42:4d:0:0:0:0:209] failed: Network is unreachable (connect failed)
	at org.apache.maven.repository.internal.DefaultModelResolver.resolveModel(DefaultModelResolver.java:160)
	at org.apache.maven.repository.internal.DefaultModelResolver.resolveModel(DefaultModelResolver.java:204)
	at org.apache.maven.model.building.DefaultModelBuilder.readParentExternally(DefaultModelBuilder.java:1009)
	at org.apache.maven.model.building.DefaultModelBuilder.readParent(DefaultModelBuilder.java:801)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:327)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:243)
	at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.loadPom(DefaultArtifactDescriptorReader.java:284)
	... 11 more
Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: The following artifacts could not be resolved: com.google.inject:guice-parent:pom:5.1.0 (absent): Could not transfer artifact com.google.inject:guice-parent:pom:5.1.0 from/to https://repo1.maven.org/maven2 (https://repo1.maven.org/maven2): Connect to repo1.maven.org:443 [repo1.maven.org/199.232.192.209, repo1.maven.org/199.232.196.209, repo1.maven.org/2a04:4e42:4c:0:0:0:0:209, repo1.maven.org/2a04:4e42:4d:0:0:0:0:209] failed: Network is unreachable (connect failed)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:473)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts(DefaultArtifactResolver.java:261)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifact(DefaultArtifactResolver.java:243)
	at org.apache.maven.repository.internal.DefaultModelResolver.resolveModel(DefaultModelResolver.java:158)
	... 17 more
Caused by: org.eclipse.aether.transfer.ArtifactTransferException: Could not transfer artifact com.google.inject:guice-parent:pom:5.1.0 from/to https://repo1.maven.org/maven2 (https://repo1.maven.org/maven2): Connect to repo1.maven.org:443 [repo1.maven.org/199.232.192.209, repo1.maven.org/199.232.196.209, repo1.maven.org/2a04:4e42:4c:0:0:0:0:209, repo1.maven.org/2a04:4e42:4d:0:0:0:0:209] failed: Network is unreachable (connect failed)
	at org.eclipse.aether.connector.basic.ArtifactTransportListener.transferFailed(ArtifactTransportListener.java:44)
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector$TaskRunner.run(BasicRepositoryConnector.java:417)
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector.get(BasicRepositoryConnector.java:260)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.performDownloads(DefaultArtifactResolver.java:537)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:449)
	... 20 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to repo1.maven.org:443 [repo1.maven.org/199.232.192.209, repo1.maven.org/199.232.196.209, repo1.maven.org/2a04:4e42:4c:0:0:0:0:209, repo1.maven.org/2a04:4e42:4d:0:0:0:0:209] failed: Network is unreachable (connect failed)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:156)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:376)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:393)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:186)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.ServiceUnavailableRetryExec.execute(ServiceUnavailableRetryExec.java:85)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)
	at org.eclipse.aether.transport.http.HttpTransporter.execute(HttpTransporter.java:485)
	at org.eclipse.aether.transport.http.HttpTransporter.implGet(HttpTransporter.java:435)
	at org.eclipse.aether.spi.connector.transport.AbstractTransporter.get(AbstractTransporter.java:64)
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector$GetTaskRunner.runTask(BasicRepositoryConnector.java:482)
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector$TaskRunner.run(BasicRepositoryConnector.java:414)
	... 23 more
Caused by: java.net.ConnectException: Network is unreachable (connect failed)
	at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:412)
	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:255)
	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:237)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.base/java.net.Socket.connect(Socket.java:615)
	at org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:368)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
```